src/cow-react/modules/limitOrders/containers/LimitOrdersConfirmModal/index.tsx

```diff
@@ -24,6 +24,7 @@ import { executionPriceAtom } from '@cow/modules/limitOrders/state/executionPric
 import { limitRateAtom } from '@cow/modules/limitOrders/state/limitRateAtom'
 import { useHandleOrderPlacement } from '@cow/modules/limitOrders/hooks/useHandleOrderPlacement'
 import { partiallyFillableOverrideAtom } from '@cow/modules/limitOrders/state/partiallyFillableOverride'
+import { useFlags } from 'launchdarkly-react-client-sdk'
```

Comment:
ubernit, but should we just hide the library behind an util with our feature flags and less dependencies. 

This allows us to not depend in darkly and define there our flags with the right types,

Reply:
Where should this live?

`cow-react/api/featureFlags/` for example?